docs/sources/write/tooling-and-workflows/index.md

```diff
@@ -301,7 +318,8 @@ git switch main
 git merge --ff-only
 ```
 
-After switching branches, git also lets you know if your local copy needs updating ("Your branch is behind 'origin/main'..."). The `--ff-only` flag means "fast-forwarding", which simply applies the changes to the end of your local copy because your local copy has no conflicting changes.
+After switching branches, git also lets you know if your local copy needs updating with the message _Your branch is behind 'origin/main'_.
```

Comment:
```suggestion
After switching branches, Git also lets you know if your local copy needs updating with the message _Your branch is behind 'origin/main'_.
```

Should we be referring to Git with initial caps? For reference: https://git-scm.com/ uses Git instead of git.

I only tagged this one instance... it's lowercase git throughout this topic (and others).

Reply:
Good call out. I think if it is _Git_ in their documentation, it should be _Git_ in ours. I'll add a Vale rule.